lutris/runners/wine.py

```diff
@@ -554,6 +588,24 @@ class wine(Runner):
                 "Which audio backend to use.\n" "By default, Wine automatically picks the right one " "for your system."
             ),
         },
+        {
+            "option": "Graphics",
+            "label": _("Graphics driver"),
+            "type": "choice",
+            "advanced": True,
+            "choices": [
+                (_("Auto"), "auto"),
+                ("Wayland", "wayland"),
+                ("X11", "x11"),
+            ],
+            "default": "auto",
+            "warning": _get_wine_wayland_warning,
+            "help": _(
+                "Which graphics backend to use.\n"
+                "By default, Wine automatically picks the right one "
+                "for your system."
```

Comment:
This help comment may be a little optimistic. Does Wine auto-detect here anything really?

Reply:
It depends on the configuration. If DISPLAY is reset then winewayland is used by default anyway, similarly if the X11 backend fails to initialize (probably in environments without Xwayland).

Reply:
I guess that's fair enough then.